src/test/java/darkdata/web/AdvisoryControllerTest.java

```diff
@@ -0,0 +1,151 @@
+import darkdata.DarkDataApplication;
+import darkdata.service.RecommendationService;
+import darkdata.web.api.RecommendationRequest;
+import darkdata.web.api.RecommendationResponse;
+import darkdata.web.api.candidate.CandidateWorkflow;
+import org.apache.commons.io.IOUtils;
+import org.hamcrest.Matchers;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.SpringApplicationConfiguration;
+import org.springframework.core.io.Resource;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import java.nio.charset.Charset;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.DoubleSummaryStatistics;
+import java.util.List;
+
+import static org.hamcrest.Matchers.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyList;
+import static org.mockito.Matchers.anyListOf;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+
+
+/**
+ * @author anirudhprabhu
+ */
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringApplicationConfiguration(classes = DarkDataApplication.class)
+@WebAppConfiguration("server.port=0")
+public class AdvisoryControllerTest {
+
+    private MediaType contentType = new MediaType(MediaType.APPLICATION_JSON.getType(),
+            MediaType.APPLICATION_JSON.getSubtype(),
+            Charset.forName("utf8"));
+
+    private MockMvc mockMvc;
+
+
+    @Mock
+    private RecommendationService recommendationService;
+
+    @InjectMocks
+    private AdvisoryController advisoryController;
+
+    @Value("classpath:json/request.json")
+    private Resource request;
+
+    @Value("classpath:json/response.json")
+    private Resource response;
+
+    @Autowired
+    private ObjectMapper mapper;
+
+
+
+//    @Autowired
+//    private List<CandidateWorkflow> candidates;
+
+    @Before
+    public void setup() throws Exception {
+        MockitoAnnotations.initMocks(this);
+        this.mockMvc = MockMvcBuilders.standaloneSetup(advisoryController).build();
+    }
+
+    @Test
+    public void testAdvisorGetForStatus() throws Exception {
+        mockMvc.perform(get("/advisor/status"))
+                .andExpect(status().isOk())
+                .andExpect(content().string("I am Up."));
+    }
+
+    @Test
```

Comment:
Let's clean-up this test by removing the commented out code (leave the TODO comments in)